tests/conftest.py

```diff
@@ -68,6 +68,7 @@ def vhl_gene_context():
                     "vega:OTTHUMG00000128668",
                     "ccds:CCDS2598",
                     "ena.embl:L15409",
+                    "iuphar:3204",
```

Comment:
not relevant to this PR, but shouldn't this be `iuphar.receptor`?

Reply:
I'm not sure. Didn't you say a while ago that this shouldn't be captured in gene?

Reply:
Also, this is all HGNC provides about IUPHAR: `"iuphar": "objectId:2465"`